resources/lang/en/snippet_collections.json

```
@@ -112,21 +115,33 @@
 
       ],
       "forest": [
+        ["the endess crashing of trees"]
 
       ],
       "plains": [
+        ["the rustling of grass in the wind"],
+        ["the drilling of a woodpecker"]
 
       ],
       "beach": [
+        ["the crashing of waves along the shore"],
+        ["a gull's cry"]
 
       ],
       "mountainous": [
+        ["the whistling wind through mountaintops"],
+        ["the roar of a cougar"]
 
       ],
       "wetlands": [
+        ["the endless croaking of frogs"],
+        ["the sound of squelching mud"]
 
       ],
       "desert": [
+        ["the rattling of a snake"],
+        ["the hiss of a snake"],
```

```suggestion
        ["the rattling of a snake, "the hiss of a snake"],
```
Since these both mention a snake, let's put them in the same list. This way, we won't have both of these being used at the same time.